src/components/VideoItems.css

```diff
@@ -1,18 +1,41 @@
 .video-item {
+  font-family: sans-serif;
```

Comment:
You globally set `font-family` in the `src/index.css`, so it can be removed from here?